cocos/platform/ios/CCDirectorCaller-ios.mm

```diff
@@ -137,7 +138,10 @@ -(void) doCaller: (id) sender
         
         [EAGLContext setCurrentContext: cocos2dxContext];
 
-        CFTimeInterval dt = ((CADisplayLink*)displayLink).timestamp - lastDisplayTime;
+        CFTimeInterval dt = 0;
+        if (lastDisplayTime != 0) {
+            dt = ((CADisplayLink*)displayLink).timestamp - lastDisplayTime;
+        }
```

Comment:
can we initialize `lastDisplayTime ` to 1/60? Then, we don't have to check it every frame, it is good for performance.